lib/task_bunny/message.ex

```diff
@@ -113,4 +113,36 @@ defmodule TaskBunny.Message do
     |> Enum.map(fn ({_q, count}) -> count end)
     |> Enum.max
   end
+
+  # Turns a job(module/atom) into a String
+  #
+  # So:
+  #   Jobs.Example => "Jobs.Example"
+  def job_to_string(job) do
+    job
+    |> Atom.to_string
+    |> String.trim_leading("Elixir.")
+  end
+
+  # Turns Strings into job(module) names
+  #
+  # So:
+  #   "Elixir.Jobs.Example" => Jobs.Example
+  #   "Jobs.Example" => Jobs.Example
+  #   "elixir.jobs.example" => Jobs.Example
+  #   "jobs.example" => Jobs.Example
+  def string_to_job(job) do
+    job
+    |> String.split(".")
+    |> Enum.map(&capitalize/1)
+    |> Enum.map(&String.to_atom/1)
+    |> Module.concat
+  end
+
+  # Capitialize the first letter, but leave other capital letters alone
+  defp capitalize(string) do
```

Comment:
Could you use [String.capitalize/1](https://hexdocs.pm/elixir/String.html#capitalize/1) ?